### ISSUE TITLE
[Bug] Cannot access partition from Linux

### ISSUE THREAD

**Issue by WithPrecedent** [Zettlr/Zettlr#3345]
### Description

I have a dual boot laptop with Windows 11 and Q4OS (a Debian distro with KDE) installed. I store all of my data, including markdown files, in a shared NFTS partition. With Zettlr installed in Windows, the shared partition shows up when I try to open files or workspaces. However, in Linux, I cannot access the shared partition because it is not listed (including in "other locations"). The partition is mounted and is accessible from the file manager (Dolphin) and in other apps (eg VS Code). 

### Reproducing

Create an NFTS partition, store files there, and try to access it from a Linux (Debian) installation of Zettlr. 

### Zettlr Version

Stable (most recent version)

### Specify version

2.2.0

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Q4OS

**Comment by nathanlesage** [comment]
So the "Open" dialog does not include the partition, did I understand that correctly?

If so, I'm not quite sure what I could do about that. Zettlr doesn't control the Open dialog, it's literally whatever your operating system offers :S

**Comment by WithPrecedent** [comment]
Thanks for your help.
Could it be that Zettlr is asking for a different, incomplete dialog from the system? I've attached a screenshot that shows the Open dialogs for VS Code, Typora, and Zettlr (in that order). Only Zettlr fails to include the "Shared" partition when I click "Other Locations."
![Screenshot_20220325_132601](https://user-images.githubusercontent.com/42685088/160180468-ad3f5157-0929-4019-9825-c26d8d3ef65c.png)

**Comment by nathanlesage** [comment]
Mh curious. I just double checked both the Electron documentation as well as the VSCode source code, but both indicate that I'm instantiating the open dialog properly.

However, I notice that the open dialog by Zettlr appears to be showing the default application icon, not the Zettlr one, which might indicate a broken `.desktop`-file. Could it be that there are some application safeguards in place that prevent an application with some possibly malformed settings to being able to access most of what's on your computer?

Also, I just noticed that you're not on the latest patch. Would you mind updating to 2.2.4 and trying it again? Maybe it's as simple as that…?

**Comment by WithPrecedent** [comment]
Thanks. The Discover Software Center and app (through Help - Check for updates) both showed no updates. So, I tried using "sudo apt upgrade" but it couldn't find the app. That made me wonder if there was something wrong with the installation.
So, I uninstalled Zettlr, rebooted, downloaded the 'deb' file, and installed through "sudo apt install ./Zettlr-*.deb." Now, everything works. 
Since this was a recent, clean install of the linux distro, I wonder if there is something going on with the version of Zettlr in the Software Center or if it isn't installing right. I also noticed that the icon for Zettlr is now consistent with my overall KDE theme (whereas it wasn't before). I don't want to go test downloading throught the Sofrware Center again since everything on my machine is now working. But I thought I'd mention it as something to keep an eye on.
Thanks for your help. I'm glad to be able to use the app in Linux!

**Comment by nathanlesage** [comment]
Wait … Zettlr is in a software center?!

Which repositories does your software center tap into? I couldn't find anything useful on the Q4OS website. Would be good to know which software center it is, because I'd argue it's maybe an outdated version.

**Comment by nathanlesage** [comment]
Oh, but closing this issue now :D But please continue to comment here, I see the notifications so we can continue investigate this!

**Comment by WithPrecedent** [comment]
I have flatpaks enabled. So, the version I found through the Software Center looks to be from flathub (since that's the only one showing up right now beside the one I have installed). When I look at it now, it is labeled as version 2.2.4. But, given my problems above, maybe there is some problem with updating (and identifying available updates) from flathub. Let me know if you want me to look at anything else.
